src/platform/nrfconnect/wifi/WiFiManager.h

```diff
@@ -178,7 +178,7 @@ class WiFiManager
     static_assert(kConnectionRecoveryMinIntervalMs < kConnectionRecoveryMaxIntervalMs);
     static_assert(kConnectionRecoveryJitterMs <= kConnectionRecoveryMaxIntervalMs);
 
-#if CHIP_DEVICE_CONFIG_ENABLE_SED
+#if defined(CHIP_DEVICE_CONFIG_ENABLE_SED) && CHIP_DEVICE_CONFIG_ENABLE_SED
```

Comment:
I would expect this to be defined in some relevant config header (probably `platform/CHIPDeviceConfig.h`, see below).  So:

1.  Is that being done?  If not, it should be. 
2. Is this file including that config header?

Looking at where this macro is defined, we have `src/platform/Zephyr/CHIPDevicePlatformConfig.h` and the  `CHIPProjectConfig.h` of a bunch of examples.  Looks like all the examples in question set `chip_device_project_config_include = "<CHIPProjectConfig.h>"` so all of those defines would in fact land in `src/include/platform/CHIPDeviceConfig.h`.  If that header is then not defining a default, that should be fixed, and this file should include that header.

Reply:
Reworked:
defined CONFIG_CHIP_THREAD_SSED which is Boolean Kconfig macros is defined as true or not defined at all:
```
#ifndef CONFIG_CHIP_THREAD_SSED
#define CONFIG_CHIP_THREAD_SSED 0
#endif
```